tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md

```diff
@@ -20,9 +20,9 @@ In this tutorial mission, setup and use these services to automate and achieve o
 
 ---
 
-[ACCORDION-BEGIN [Step 1: ](Check Entitlements)]
+[ACCORDION-BEGIN [Step 1: ](Setup services in your account using Recipe)]
 
-To create service instances for business process management services, you need to ensure that you have the right set of entitlements in your trial account.
+You will use the **recipe** to automatically setup of workflow, business rules, and process visibility services in your account.
```

Comment:
Please rephrase.

- for automatic setup of...
or
- to automatically set up the workflow...

Reply:
Done